lib/puppet/pops/binder/lookup.rb

```diff
@@ -137,7 +137,14 @@ def self.search_for(scope, type, name, options)
    end
   end
 
-  # This is the method called from the puppet/parser/functions/lookup.rb
```

Comment:
This method doesn't provide access to anything that wasn't public before. The reason I broke it out into a separate method was purely stylistic. I guess {{search_path}} and some other methods are broken out for the same reason? Perhaps they all should be private?